language/tools/move-mv-llvm-compiler/src/stackless/translate.rs

```diff
@@ -168,13 +170,15 @@ impl<'mm, 'up> ModuleContext<'mm, 'up> {
 
         self.declare_structs();
         self.llvm_module.declare_known_functions();
+
+        // Declaring functions will populate list `expanded_functions` containing all
+        // concrete Move functions and expanded concrete instances of generic Move functions.
         self.declare_functions();
 
-        for fn_env in self.env.get_functions() {
-            if fn_env.is_native() {
-                continue;
-            }
-            let fn_cx = self.create_fn_context(fn_env, &self);
+        for fn_qiid in &self.expanded_functions {
```

Comment:
Just out of curiosity what `qiid` stands for?

Reply:
QualifiedInstId. It is common in the "model" to use qid for QualifiedId, I'm doing same for QualifiedInstId. BTW, the second version is just a QualifiedId with an additional type parameter vector attached.